spanner/setup.py

```diff
@@ -59,7 +59,7 @@
 
 setup(
     name='google-cloud-spanner',
-    version='0.23.1',
+    version='0.23.2',
```

Comment:
Do the GAPIC deps not also need to be revved to include `lib_name`/`lib_version`?